tutorials/terraform/sqlserver-migration/to-mms-with-bucket.tf

```
@@ -0,0 +1,98 @@
+# Infrastructure for Yandex Cloud Managed Service for SQL Server.
+#
+# RU: https://cloud.yandex.ru/docs/managed-sqlserver/tutorials/migration-with-bucket
+# EN: https://cloud.yandex.com/en/docs/managed-sqlserver/tutorials/migration-with-bucket
+#
+# Set the configuration of the Managed Service for SQL Server cluster:
+locals {
+  folder_id          = "" # Your folder ID.
+  sql_server_version = "" # Set the SQL Server version. It must be the same or higher than the version in the source cluster.
+  db_name            = "" # Set a database name.
+  username           = "" # Set a user name.
+  password           = "" # Set a user password.
+  storage_sa_id      = "" # Service account ID for creating a bucket in Object Storage.
```

Раз уж везде с глагола начинаем, давай и тут:
```suggestion
  storage_sa_id      = "" # Set a service account ID for creating a bucket in Object Storage.
```

---

Тут штука в том, что все остальные поля мы именно задаем, а здесь и в `folder_id` указываем существующее. Здесь надо указать ID существующего аккаунта.

---

Тогда так: 
# Set the service account ID for creating a bucket in Object Storage.